source/fundamentals/atlas-vector-search.txt

```diff
@@ -0,0 +1,152 @@
+.. _csharp-atlas-vector-search:
```

Comment:
Do we want to mention is some other section that we support Binary Vector format now? (Serialization?)
It's significantly faster for serializing arrays of a certain types, so might interesting to users.
Technically it can be used without vector search, just to store data. So maybe a brief not that links to Vector Search page.

Reply:
I think it's a good idea, but I couldn't really find anywhere that it would fit without being out of place in our current docs. We do have it mentioned in the What's New, but we don't really have anywhere else that points out what kinds of data can be serialized into another